code/game/jobs/job/syndicate_jobs.dm

```diff
@@ -23,27 +23,28 @@
 	back = /obj/item/storage/backpack
 	head = /obj/item/clothing/head/beret
 	gloves = /obj/item/clothing/gloves/combat
-	shoes = /obj/item/clothing/shoes/combat
-	mask = /obj/item/clothing/mask/cigarette/cigar/havana
-	belt = /obj/item/gun/projectile/automatic/pistol/deagle/camo
+	shoes = /obj/item/clothing/shoes/magboots/syndie/advance
```

Comment:
i aint gonna lie, these magboots look kinda silly. Do you think you can let them have either the magboots or backup shoes in their bag?

Reply:
Honestly, I wish the Advanced versions of the Syndie Magboots were the same black color scheme as the other "advanced/elite" items.

Would match the getup so much better.

Reply:
I have now made #22968 to make this less awful of a color match.